imap_processing/tests/spice/test_geometry.py

```diff
@@ -306,3 +308,49 @@ def test_instrument_pointing(furnish_kernels):
             et, SpiceFrame.IMAP_HI_90, SpiceFrame.ECLIPJ2000, cartesian=True
         )
         assert ins_pointing.shape == (3, 3)
+
+
+def test_cartesian_to_spherical():
```

Comment:
I would love to see this test bulked up by using a meshgrid of `x, y, z` values in the range `[-1, 1]` with steps of something like 0.05 so that all directions are covered. Results can be checked against the spiceypy function `recsph`. Note spice defines phi in the range `[-pi, pi]`.